floris/tools/floris_interface.py

```diff
@@ -760,18 +754,18 @@ def get_farm_power(
             # Default to equal weighing of all turbines when turbine_weights is None
             turbine_weights = np.ones(
                 (
-                    self.floris.flow_field.n_wind_directions,
-                    self.floris.flow_field.n_wind_speeds,
+                    self.floris.flow_field.n_findex,
                     self.floris.farm.n_turbines
                 )
             )
+
+        #TODO: PF Confirm this is correct?
```

Comment:
Is this still an open question?

Reply:
Good catch, this is resolved, I wrote a test for it to be sure